SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -217,8 +222,11 @@ void USpatialWorkerConnection::QueueLatestOpList()
 
 void USpatialWorkerConnection::ProcessOutgoingMessages()
```

Comment:
Does this only ever get called one per tick? Or are there certain circumstances under which it's called more than that?

Reply:
We call this roughly every 1ms on a background thread - I suspect the OutgoingMessagesQueue is empty most of the time until there is active game thread work doing GDK stuff so it could do with some improving

We have a config to change this to run on the main thread and flush after RPCs and replication but currently the default is to call this in a loop every 1ms. See background thread here https://github.com/spatialos/UnrealGDK/pull/1997/files#diff-84dd8b2dbd454b9ba83b56c726a732d6L182